### PR TITLE
Add AWS serverless icons collection

### PR DESCRIPTION
**Pull request by stojanovic**
Adding a collection of AWS Serverless services icons. The collection contains the following services:

- AWS Lambda
- Amazon AppSync
- Amazon API Gateway
- Amazon EventBridge
- Amazon DynamoDB

Preview:
aws-serverless

### CODE REVIEW

**Comment by dwelle** [comment]
Looks good. You may consider putting a semi-opaque white shape above the cross-hatch bg to make the logos more legible:

image

https://excalidraw.com/#json=4884205674692608,_DVr9h9NUxnnue6LHE0uWA

**Comment by stojanovic** [comment]
@dwelle thanks! Yes, that's a good idea. I guess putting a slightly lighter color would have the same effect, and it would still work fine in the dark mode.

I'll try to play a bit more with colors when I add a few additional icons in the next few days.

**Comment by lipis** [comment]
Or another idea about is to add `00-ff` (opacity) to the background color of the icons.. For example `#e64980` can become `#e6498088`..

**Comment by stojanovic** [comment]
@lipis good idea, let me try that

**Comment by stojanovic** [comment]
This looks a bit better, thanks!

aws-serverless
aws-serverless-dark-mode

File: libraries.json [hunk 1]
```diff
@@ -336,5 +336,18 @@
     "source": "spfr/lo-fi-wireframing-kit.excalidrawlib",
     "preview": "spfr/lo-fi-wireframing-kit.png",
     "date": "2021-02-10"
+  },
+  {
+    "name": "AWS Serverless Icons",
+    "description": "A collection of icons of AWS serverless services",
```
Comment (lipis) [inline libraries.json:342]: can we name the services so it will be easier to search with `cmd+f`

Comment (stojanovic) [reply libraries.json:342]: @lipis Can I name each icon? If not should I just edit the description to include service names?

Comment (lipis) [reply libraries.json:342]: I was thinking just a comma separated thingy in the description.

**Comment by lipis** [comment]
aws-serverless

Can you update also the preview with this one?

**Comment by stojanovic** [comment]
@lipis I updated both the description and the preview image.

File: libraries.json [hunk 2]
```diff
@@ -336,5 +336,18 @@
...
+    "description": "A collection of icons of AWS serverless services. This collection contains the icons of the following services: AWS Lambda, Amazon API Gateway, AWS AppSync, Amazon DynamoDB, and Amazon EventBridge.",
+    "authors": [
+      {
+        "name": "Slobodan Stojanović",
+        "url": "https://slobodan.me"
+      }
+    ],
+    "source": "slobodan/aws-serverless.excalidrawlib",
+    "preview": "slobodan/aws-serverless.png",
+    "date": "2021-02-11"
```
Comment (lipis) [inline libraries.json:351]:
```suggestion
    "date": "2021-02-12"
```

**Comment by lipis** [comment]
Thanks @stojanovic! I guess we could add more icons later :)

**Review by lipis — APPROVED** [review]

**Review by ad1992 — APPROVED** [review]
🎉 🎉

**Comment by stojanovic** [comment]
I'll add more icons at some point, I started with a few that I need for my next workshop :)

**Pull request merged**